wsmaster/che-core-api-workspace/src/main/resources/schema/devfile.json

```diff
@@ -145,6 +145,14 @@
                 "examples": [
                   "349d3ad"
                 ]
+              },
+              "sparseCheckoutDir": {
+                "type": "string",
+                "description": "The directory which will be created with sparse checkout.",
```

Comment:
For me, the description seems not very clear.
I may assume that some specified directory will be created and my project will be imported inside.

Maybe (maybe not) it would be better to reuse something from http://schacon.github.io/git/git-read-tree.html
> "Sparse checkout" allows to sparsely populate working directory